Mailbox/Mailbox.cs

```diff
@@ -1,4 +1,37 @@
-﻿namespace Mailbox
+﻿using System;
+
+namespace Mailbox
 {
+    public class Mailbox
+    {
+        public Size Size { get; set; }
+        public (int x, int y) Location { get; set; }
+        public Person Owner { get; set; }
+
+        public Mailbox(Size size, ValueTuple<int,int> location, Person owner)
```

Comment:
ValueTuple syntax can be simplified
```suggestion
        public Mailbox(Size size, (int,int) location, Person owner)
```